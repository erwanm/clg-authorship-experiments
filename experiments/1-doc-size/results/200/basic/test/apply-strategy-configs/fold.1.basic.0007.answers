1903-us-tjoas 1854-hbs-smofl	0.609361
1895-wta-alae 1906-mt-cfma	0
1891-fmc-ttf 1881-haj-fcbtp	0.901632
1872-haj-ptf 1906-rwc-ttolp	0.393303
1898-hj-ttm 1879-hj-h	0.41115
1874-wta-tcw 1900-haj-adoh	0.708953
1878-hj-te 1882-lma-ajsb-v6	0.328822
1883-hj-rh 1898-hj-itc	0.507891
1885-hj-gr 1910-us-sts	0.529858
1914-us-sm 1897-ewaoc-tdof	0.433078
1896-espw-cfal 1887-espw-jtf	0.787492
1887-espw-tgb 1900-espw-wtg	1
1894-cdw-tgh 1907-fmc-tdr	0.49814
1900-fmc-itpotk 1905-fmc-fm	0.466974
1889-fmc-g 1898-haj-tyb	0.956738
1890-hj-ttm 1904-hj-tgb	0.766735
1878-hbs-pp 1870-hbs-tbc	0.366564
1901-us-km 1921-us-tbol	0.720784
1883-hj-rh-v2 1872-wta-lb	0.252616
1866-haj-ccc-ia 1871-hbs-mwai	1
1880-hj-c 1885-hj-altof	0.282928
1898-fmc-ari-v2 1893-fmc-kl-v2	0.002089
1918-rwc-tlg 1863-wta-isao	0.268528
1884-ab-sbts 1898-us-apom	0.286681
1889-cfw-jl 1915-rwc-p	0
1872-mt-ri 1895-haj-tdw	1
1902-fmc-c 1883-fmc-tl	0.704395
1859-hbs-tmw 1887-fmc-pp	0.438164
1884-wta-Sac 1882-wta-aa	0.40766
1871-haj-tt 1876-wta-vao	0
1913-es-tm 1911-rwc-tgr	0.448269
1905-es-tpoe 1907-es-tlotg	0.046022
1899-haj-mmv 1866-haj-hf	0.822272
1891-fmc-do 1876-cdw-itl	0.529977
1892-fmc-pg 1883-fmc-dc	0.995383
1917-ab-bn 1922-us-tcmc	0.535257
1889-cdw-sths 1869-espw-mwag	0.401014
1902-hj-twotd 1896-hj-tsop	0.942334
1888-haj-bb 1899-haj-ra	0.614016
1913-ew-tcotc 1868-haj-rd	1
1868-espw-tga 1919-rwc-tsos	0.270994
1900-haj-ainy 1919-ew-m	0.609125
1881-mt-tpatp 1899-haj-aoatb	0.77901
1891-fmc-k 1893-fmc-tr	0.058778
1881-hj-tpoal 1890-fmc-acmr	0.338074
1869-haj-mtmb 1903-haj-bba-ph	1
1885-fmc-atoalp 1905-fmc-wso	0.474269
1868-wta-mob 1870-wta-faf	0.737865
1871-hbs-pawt 1904-rwc-isotu	0.528347
1875-cfw-cn 1864-lma-m	1
1914-mh+-mhce-nf 1908-fmc-tp	0.118639
1893-fmc-md 1895-fmc-ajs	0.331433
1864-wta-tsb 1883-wta-sh	0.982642
1900-rwc-taoamm 1910-rwc-tgm	1
1903-fmc-thor 1887-fmc-s	0.873327
1884-mt-taohf 1874-mt-lotm	0
1907-espw-tldup 1871-espw-tsp	0.121123
1866-espw-gcj 1882-haj-bn	0.622059
1883-haj-tbb 1888-es-ttatv	0.085231
1908-fmc-s 1908-fmc-tws	0.279326
1889-wta-wtel 1911-rwc-tcl	0.332458
1904-rwc-sm 1886-hj-pc	0.075176
1918-us-jh 1897-ew-tdof	1
1872-lma-ss 1867-lma-tagomtt	0.3023
1910-ew-tomag 1889-es-atg	0.989369
1889-espw-tsfi 1895-espw-tsojc	0.21634
1905-haj-fftf 1870-haj-rar	1
1882-espw-dz 1909-mt-isd	0.992928
1907-ew-tfott 1896-hj-toh	0.234399
1868-wta-dad 1885-wta-sts	0.244533
1909-rwc-tdm 1897-ab-tdohy	0
1891-haj-tetb 1868-haj-faf	1
1893-haj-dtn 1866-haj-tcw	1
1869-wta-dtr 1865-wta-waw	1
1891-cdw-oi 1873-hbs-wish	0.859962
1879-hj-dm 1903-hj-ta	1
1874-cfw-tac 1882-fmc-mi	0.693976
1893-espw-dm 1896-ab-boat	0.726172
1896-espw-tsasa 1894-espw-asl	1
1879-haj-tym 1869-haj-pjb	0.893837
1920-us-1tsoap 1913-us-dg	0.619122
1917-es-ow-sh 1873-hbs-pl	0.79422
1855-wta-aa 1873-wta-tyc	0.904544
1879-haj-ttb 1894-fmc-wg	0.373727
1898-fmc-vc 1895-fmc-t	0.468288
1867-espw-gyatgc 1901-espw-a	0.44671
1896-fmc-c 1885-fmc-zamc	0.343408
1855-hbs-tmf 1866-espw-gb	0.493283
1897-fmc-aroy 1884-fmc-aap	1
1883-espw-btg 1911-hj-to	0.467824
1862-hbs-tpooi 1875-hbs-waon	1
1883-fmc-ars 1885-fmc-tcotk	0.466842
1886-hbs-tsw 1854-hbs-smofl-v2	1
1903-ew-ivatg 1878-cdw-hikab	0.52023
1858-wta-pap 1897-haj-fof	1
1902-hj-twotd-v2 1888-hj-tr	0.01027
1871-hbs-ofs 1888-fmc-si	0.263834
1907-ab-rm 1901-hj-tsf	1
1882-haj-ffbts 1893-haj-ianwatgfoa	1
1908-rwc-tfl 1910-rwc-ap	0
