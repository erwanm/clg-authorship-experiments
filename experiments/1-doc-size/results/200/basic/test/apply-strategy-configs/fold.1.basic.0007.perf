0.302206	0.570200	0.530000
