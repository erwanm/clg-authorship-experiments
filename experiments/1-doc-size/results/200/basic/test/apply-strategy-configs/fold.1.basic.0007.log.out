apply-config.sh: applying config '/tmp/200/process//selected-meta-configs/fold.1.strategy-as-meta.basic.0007.model/strategy-configs/fold.1.basic.0007' to cases '/home/moreaue/dump/expe1//results/200/basic/test/apply-strategy-configs/only-cases.tmp.2Jhq8BCJ'; results written to '/home/moreaue/dump/expe1//results/200/basic/test/apply-strategy-configs/fold.1.basic.0007'
apply-config.sh: done.
